js/about/preferences.js

```diff
@@ -1316,6 +1338,16 @@ class PaymentsTab extends ImmutableComponent {
         ? <ModalOverlay title={'ledgerRecoveryTitle'} content={this.ledgerRecoveryContent} footer={this.ledgerRecoveryFooter} onHide={this.props.hideOverlay.bind(this, 'ledgerRecovery')} />
         : null
       }
+      <div className='advancedSettingsWrapper'>
+        {
+          this.props.ledgerData.get('created') && this.enabled
+          ? <Button
+            l10nId='advancedSettings'
+            className='advancedSettings whiteButton'
```

Comment:
note for reference: https://github.com/brave/browser-laptop/pull/6388/files#diff-e3eeb751016b2ce9f8278efce585a461R1314 of #6388